content/TrueCommand/TCGettingStarted/Install/InstallTCCloud.md

```diff
@@ -0,0 +1,167 @@
+---
+title: "Installing TrueCommand Cloud"
+description: "This article describes the steps for installing TrueCommand Cloud."
+weight: 10
+tags:
+- tccloudinstall
+- tcinstall
+- tccloud
+- scaletoptoolbar
+- coretoptoolbar
+---
+{{< toc >}}
+
+TrueCommand is versatile and offers several different install options.
+TrueCommand Cloud is the preferred method for using TrueCommand since it requires no local resources or specific hardware requirements to get started!
+
+{{< hint info >}}
+Please note that a credit card number is required for Cloud Signups.
+
+Purchasing a TrueCommand license as part of an Enterprise contract provides special benefits. Contact [**iXsystems Customer Support**]({{< relref "/TrueCommand/Introduction/Support.md" >}}). Request a sign-up code from a member of the Deployment Team. This code provides the TrueCommand Cloud license at no cost for the duration of the contract.
```

Comment:
Suggestion to clarify text a little to make it obvious the sign-up code is an additional option for people who are already purchasing a TrueNAS Enterprise license. 
```suggestion
If you're purchasing TrueNAS Enterprise hardware or licenses, you might be eligible to receive an additional TrueCommand license that lasts the duration of the TrueNAS Enterprise contract. Contact [**iXsystems Customer Support**]({{< relref "/TrueCommand/Introduction/Support.md" >}}) for more details about adding TrueCommand to your TrueNAS Enterprise purchase.
```